kraken/pod_scenarios/setup.py

```diff
@@ -78,7 +78,7 @@ def container_run(kubeconfig_path, scenarios_list, config, failed_post_scenarios
             for cont_scenario in cont_scenario_config["scenarios"]:
                 # capture start time
                 start_time = int(time.time())
-                killed_containers = container_killing_in_pod(cont_scenario)
+                killed_containers = container_killing_in_pod(cont_scenario, kubecli)
```

Comment:
The logging is repetitive when running the container scenarios. For example: 
```
023-05-09 02:21:35,755 [INFO] Running container scenarios
2023-05-09 02:21:37,671 [INFO] Killing container etcd in pod etcd-ip-10-0-134-51.us-west-2.compute.internal (ns openshift-etcd)
2023-05-09 02:21:37,764 [INFO] Killing container etcd in pod etcd-ip-10-0-134-51.us-west-2.compute.internal (ns openshift-etcd)
2023-05-09 02:21:37,849 [INFO] Killing container etcd in pod etcd-ip-10-0-134-51.us-west-2.compute.internal (ns openshift-etcd)
2023-05-09 02:21:37,907 [INFO] Killing container etcd in pod etcd-ip-10-0-134-51.us-west-2.compute.internal (ns openshift-etcd)
2023-05-09 02:21:37,966 [INFO] Killing container etcd in pod etcd-ip-10-0-134-51.us-west-2.compute.internal (ns openshift-etcd)
2023-05-09 02:21:38,023 [INFO] Scenario kill etcd container successfully injected
2023-05-09 02:21:38,558 [INFO] Waiting for the specified duration: 60
```

VS

```
2023-05-09 02:26:42,431 [INFO] Running container scenarios
2023-05-09 02:26:44,188 [INFO] Killing container etcd in pod etcd-ip-10-0-165-200.us-west-2.compute.internal (ns openshift-etcd)
2023-05-09 02:26:44,318 [INFO] Scenario kill etcd container successfully injected
2023-05-09 02:26:44,926 [INFO] Waiting for the specified duration: 60
```